packages/lab/src/dialog/DialogHeader.tsx

```diff
@@ -86,7 +87,7 @@ export const DialogHeader = forwardRef<HTMLDivElement, DialogHeaderProps>(
             {preheader && (
               <Text className={withBaseName("preheader")}>{preheader}</Text>
             )}
-            {header}
+            <H2 className={withBaseName("header")}>{header}</H2>
```

Comment:
There's a change no matter what here, the preheader is no longer part of the header. @jake-costa Do you see this as a better outcome or do you think the previous was better?

Reply:
@joshwooding good to hear the preheader is no longer part of the header. Also, I'd like to connect to go over this if you have some time this week but personally I think this approach would be better. 

Lastly, I want to add that we need to be careful about assigning h4 to the overlay, given it is also technically a modal it should probably be an h2 as well.

Reply:
updated h4s in our stories for h2 that look like h4s

Reply:
@Fercas123 can you set up a call with Jake please.

Reply:
updated the h2 to wrap preheader again and swapped overlay header to do the same